plugins/module_utils/crypto/cryptography_openssh.py

```diff
@@ -0,0 +1,484 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright: (c) 2021, Andrew Pantuso (@ajpantuso) <ajpantuso@gmail.com>
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+from base64 import b64encode, b64decode
+from getpass import getuser
+from socket import gethostname
+
+try:
+    from .basic import (
+        HAS_CRYPTOGRAPHY,
+        CRYPTOGRAPHY_HAS_ED25519,
```

Comment:
These symbols are always defined, there's no need to catch an `ImportError` here.